packages/panzer/mini-em/example/BlockPrec/main.cpp

```diff
@@ -139,7 +139,6 @@ int main_(Teuchos::CommandLineProcessor &clp, int argc,char * argv[])
     int numReps = 0;
     std::string linAlgebra = "Tpetra";
     use_stacked_timer = true;
-    Teuchos::CommandLineProcessor clp;
```

Comment:
@cgcgcg, this `clp` was shadowing the `clp` being passed in as a parameter.  Is this the right fix here, or should the function signature be amended?

Reply:
@jmgate This seems like the correct fix. Curious, that things still seemed to be working..